corehq/toggles/__init__.py

```diff
@@ -2012,17 +2012,6 @@ def _commtrackify(domain_name, toggle_is_enabled):
     parent_toggles=[INBOUND_SMS_LENIENCY]
 )
 
-CHANGE_FORM_LANGUAGE = StaticToggle(
-    'change_form_language',
-    'USH: Allow user to change form language in web apps',
-    TAG_CUSTOM,
-    namespaces=[NAMESPACE_DOMAIN],
-    description="""
-    Allows the user to change the language of the form content while in the form itself in Web Apps
-    """,
-    help_link="https://confluence.dimagi.com/display/saas/Change+Form+Language"
```

Comment:
Are there any public-facing docs that should be updated?

Reply:
Should confluence page for FF be archived?

Reply:
This page should be archived, yes, but the content of it should probably be moved to a public page.